lib/base64_attach.rb

```diff
@@ -0,0 +1,10 @@
+module Base64Attach
+  extend self
+
+  def from_base64(data:, filename:)
```

Comment:
After some discussion with @santib, since we could expect this module to be included into a controller, I'd like to find a better name for this method. The name `from_base64` is bit too generic, and we would probably hesitate to change these things after it's on the upstream.

I think `attachment_from_base64` could be a good fit, what do you people think?

In addition, it seems like reek doesn't like the name ending with `64`, can we remove that rule? Ruby itself uses names ending with numbers, so I don't think it's something bad.

Reply:
I think reek doesn't like names to end in numbers so you don't do things like `user1`, `user2`, to differentiate params so I wouldn't remove that rule completely but this specific case is different. I would ignore only this case (Just my opinion)

Reply:
I don't really see any problem with `user1` and `user2`, it sounds like depending on the context, could be correct names. Why can we use `first_user`, `second_user`, `user_one`, `user_two`, but not those? I think those restrictions are unnecessary.

Anyways, adding an exception for this method could be enough for now.

Reply:
I'm good both with the name `attachment_from_base64` and excluding the reek check for this method.